acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/GenesisStateAcceptanceTest.java

```diff
@@ -39,4 +45,36 @@ public void shouldCreateTheSameGenesisState() throws Exception {
     // state because they processed the same deposits from the same ETH1 chain.
     lateJoinTeku.waitUntilInSyncWith(firstTeku);
   }
+
+  @Test
+  public void shouldCreateGenesisFromPartialDeposits() throws Exception {
+    final BesuNode eth1Node = createBesuNode();
+    eth1Node.start();
+    int numberOfValidators = 4;
+
+    final Eth1Address eth1Address = Eth1Address.fromHexString(eth1Node.getDepositContractAddress());
+    final Credentials eth1Credentials = Credentials.create(eth1Node.getRichBenefactorKey());
+    DepositGenerator depositGenerator =
+        new DepositGenerator(
+            eth1Node.getExternalJsonRpcUrl(),
+            eth1Address,
+            eth1Credentials,
+            numberOfValidators,
+            UInt64.valueOf(MAX_EFFECTIVE_BALANCE / 2));
+
+    depositGenerator
+        .generateKeysStream()
+        .forEach(
+            key -> {
+              // For each key send deposit twice, since these are partial deposits
+              depositGenerator.sendDeposit(key).join();
+              depositGenerator.sendDeposit(key).join();
+            });
+
+    final TekuNode teku = createTekuNode(config -> config.withDepositsFrom(eth1Node));
+    teku.start();
+    teku.waitForGenesis();
```

Comment:
Yeah as you've commented this isn't the right approach.  We want the AT to be simple and clear, but still show what steps its actually taking.  So we want the AT to come out something like:

```
@Test
  public void shouldCreateGenesisFromPartialDeposits() throws Exception {
    final BesuNode eth1Node = createBesuNode();
    eth1Node.start();
    int numberOfValidators = 4;

    final TekuDepositSender depositSender = createTekuDepositSender();
    final List<ValidatorKeyGenerator.ValidatorKeys> validatorKeys =
        depositSender.generateValidatorKeys(numberOfValidators);
    depositSender.sendValidatorDeposits(eth1Node, validatorKeys, MIN_DEPOSIT_AMOUNT);
    depositSender.sendValidatorDeposits(
        eth1Node, validatorKeys, MAX_EFFECTIVE_BALANCE - MIN_DEPOSIT_AMOUNT);

    final TekuNode teku = createTekuNode(config -> config.withDepositsFrom(eth1Node));
    teku.start();
    teku.waitForGenesis();

    teku.waitForValidators(numberOfValidators);
  }
```

Reply:
Done.